util/topgen/lib.py

```diff
@@ -353,6 +353,19 @@ def is_top_reggen(module):
         return False
 
 
+def is_reggen_only(module):
+    """Returns an indication where a particular module is NOT templated,
+       requires top level specific reggen and is NOT instantiated in the
+       top
+    """
+    if "attr" not in module:
+        return False
+    elif module["attr"] in ["reggen_only"]:
+        return True
+    else:
+        return False
```

Comment:
Equivalent and maybe simpler?
```suggestion
    return module.get('attr') == 'reggen_only'
```

Reply:
ah that is so much better. let me update the other ones to do this too.